packages/core-typings/src/IRoom.ts

```diff
@@ -74,6 +74,7 @@ export interface IRoom extends IRocketChatRecord {
 	usernames?: string[];
 	ts?: Date;
 
+	closedAt?: Date;
```

Comment:
why was this moved from `IOmnichannelGenericRoom` to `IRoom`?

Reply:
That's because I converted [this file](https://github.com/RocketChat/Rocket.Chat/blob/1e5a0650d9afd8c2649765241e14f58a40443f62/apps/meteor/imports/message-read-receipt/server/hooks.js#L14) to TS and I'd have the `"Property closedAt does not exist on type IRoom"` error if I didn't do so. Also, since any room can be closed and have this property, I guess it makes sense to move it to `IRoom`. wdyt?

Reply:
not really, only omnichannel rooms can be closed.. or am I missing something? how do you close a room that is not an omnichannel room?

Reply:
You're right, I guess I confused it with the `closedAt` field in subscriptions :grimacing: 
That's fixed.